openfermioncirq/gates/three_qubit_gates.py

```diff
@@ -136,3 +138,64 @@ def CRyxxy(rads: float) -> CYXXYPowGate:
 
 CXXYY = CXXYYPowGate()
 CYXXY = CYXXYPowGate()
+
+
+@cirq.value_equality(approximate=True)
+class CombinedCXXYYPowGate(
```

Comment:
Why is this called CombinedCXXYYPowGate? It's not clear to me from the documentation.

Reply:
`CombinedCXXYYPowGate` is to `CXXYYPowGate` as `CombinedDoubleExcitationGate` is to `DoubleExcitationGate`, in the sense that `CXXYYPowGate` corresponds to time evolution under a term (n_p a*_q a_r + hc) while `CombinedCXXYYPowGate` corresponds time evolution under a weighted sum of that term, (n_q a*_p a_r  + hc), and (n_r a*_q a_p), i.e. all 2-body terms that have exactly three unique indices.

The name is terrible and other suggestions are very welcome. One possibility is CubicInteractionGate (and similarly CombinedSwapAndZ would be QuadraticInteractionGate and CombinedDoubleExcitationGate would be QuarticInteractionGate).